runtime/interpreter/interpreter.go

```diff
@@ -4797,16 +4797,16 @@ func (interpreter *Interpreter) GetCompositeType(
 			return compositeType, nil
 		}
 	} else {
-		compositeType = interpreter.getUserCompositeType(location, typeID)
```

Comment:
I wonder if this is the real issue. You had mentioned this works correctly in `cadence-tools/test` but not working in emulator/cli? Could that be because of the way certain APIs are implemented by those? e.g. The implementation of `importResolver` (and/or some other interface methods) could be different in the two repos?

This fix also works, but trying to figure out if there's a different/bigger issue that might get buried by this fix, and might pop again later in a different form.

Reply:
That's a good point @SupunS :+1: 

I have opened a PR (https://github.com/onflow/cadence-tools/pull/195/files), which adds the relevant assertions for loading of standard library flow events. As you can see in the changed files, the `importResolver` implementation there is a NO-OP:

```go
importResolver := func(location common.Location) (string, error) {
	return "", nil
}

runner := NewTestRunner().WithImportResolver(importResolver)
```
This is probably the reason that it works only in the testing suite, and not in the actual testing framework (released build).

Since the `CompositeTypeHandler` was introduced rather recently (https://github.com/onflow/cadence/pull/2533), I am guessing that the misbehavior arose from that PR. It's rather strange that it does not work for either Emulator or the Cadence Testing Framework.

Reply:
> This is probably the reason that it works only in the testing suite, and not in the actual testing framework (released build).

yeah, that seems a possible root-cause 👍